_data/projects/ML

```diff
@@ -0,0 +1,12 @@
+name: ML
+desc: Project on Linear Regression
+site: datascience.com/highest-rated-ml-projects-on-github-694486293
```

Comment:
Is this link correct? For me it redirects to `https://www.oracle.com/corporate/acquisitions/datascience/` which doesn't seem relevant.